examples/src/main/python/mllib/naive_bayes_example.py

```diff
@@ -50,8 +61,17 @@ def parseLine(line):
     # Make prediction and test accuracy.
     predictionAndLabel = test.map(lambda p: (model.predict(p.features), p.label))
     accuracy = 1.0 * predictionAndLabel.filter(lambda (x, v): x == v).count() / test.count()
+    print('\n\tmodel accuracy %.4f\n' % accuracy)
```

Comment:
It is useful to print the accuracy. But the code could be just `print('model accuracy: %f' % accuracy)`, again, for simplicity.